bin/express

```diff
@@ -49,7 +49,7 @@ program
   .option('    --pug', 'add pug engine support', renamedOption('--pug', '--view=pug'))
   .option('    --hbs', 'add handlebars engine support', renamedOption('--hbs', '--view=hbs'))
   .option('-H, --hogan', 'add hogan.js engine support', renamedOption('--hogan', '--view=hogan'))
-  .option('-v, --view <engine>', 'add view <engine> support (ejs|hbs|hjs|jade|pug|twig|vash) (defaults to jade)')
+  .option('-v, --view <engine>', 'add view <engine> support (ejs|hbs|hjs|jade|pug|twig|vash|none) (defaults to jade)')
```

Comment:
If you're happy either way, I like 'none' best as I think it's the most clear when talking about what view engine is there (I'd be similarly happy with HTML). The correlation to `express.static()` is interesting, but I think it's marginally less understandable - I don't think I'd have a good idea what a "static view engine" was.